components/brave_wallet/browser/zcash/zcash_rpc.cc

```diff
@@ -259,6 +259,23 @@ const GURL MakeGetCompactBlocksURL(const GURL& base_url) {
   return base_url.ReplaceComponents(replacements);
 }
 
+const GURL MakeGetSubtreeRootsURL(const GURL& base_url) {
```

Comment:
We should not return values marked with const. This applies to other functions in this file
~const~ GURL